diff --git a/node_modules/@eth-optimism/core-utils/build/src/coders/sequencer-batch.js b/node_modules/@eth-optimism/core-utils/build/src/coders/sequencer-batch.js
index 55c7907..4151e16 100644
--- a/node_modules/@eth-optimism/core-utils/build/src/coders/sequencer-batch.js
+++ b/node_modules/@eth-optimism/core-utils/build/src/coders/sequencer-batch.js
@@ -3,7 +3,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 exports.decodeAppendSequencerBatch = exports.encodeAppendSequencerBatch = void 0;
 const common_1 = require("../common");
 const ethers_1 = require("ethers");
-const utils_1 = require("ethers/lib/utils");
+const utils_1 = require("ethers/utils");
 const APPEND_SEQUENCER_BATCH_METHOD_ID = 'appendSequencerBatch()';
 const appendSequencerBatch = async (OVM_CanonicalTransactionChain, batch) => {
     const methodId = utils_1.keccak256(Buffer.from(APPEND_SEQUENCER_BATCH_METHOD_ID)).slice(2, 10);
